plugins/org.jboss.tools.openshift.core/src/org/jboss/tools/openshift/core/connection/ConnectionFactory.java

```diff
@@ -30,6 +30,8 @@
  */
 public class ConnectionFactory implements IConnectionFactory {
 
+	private static final String USERDOC_DEFAULT_URL = "http://tools.jboss.org/documentation/howto/openshift3_getting_started.html";
+	private static final String SIGNUP_URL = "https://manage.openshift.com/accounts/auth/keycloak";
```

Comment:
@jeffmaury by browsing. It's by the way the url that you get redirected to when browsing to the url that alexey pointed us to. I'll change this to the url that he suggested.